db/patterns/zippyshare_widget.eno

```diff
@@ -1,6 +1,6 @@
 name: Zippyshare Widget
 category: customer_interaction
-website_url: http://www.zippyshare.com
+website_url: https://www.zippyshare.cloud/
```

Comment:
Why change it to .cloud? Wikipedia points https://zippyshare.com/

(https://en.wikipedia.org/wiki/Zippyshare)